charts/tekton-pipelines/templates/kaniko/trigger-template.yaml

```diff
@@ -52,6 +56,8 @@ spec:
           value: "$(tt.params.docker_context)"
         - name: kaniko_extra_args
           value: "$(tt.params.kaniko_extra_args)"
+        - name: repository_submodules
+          value: "$(tt.params.repository_submodules)"
```

Comment:
the only thing is that I don't understand how it is suppose to work? Tekton (our version) expects SSH key per repo and github does not allow the use of the same SSH key in different repos. So this will fail, when tekton will try to clone the repo defined as submodule, as it will use config/ssh with the ssh deploy key of the parent repo.

Reply:
It is just a pram for now, so I could set `repository_submodules` to `False` and no submodules cloning will happen during build process. Currently it resolves my issue and I don't need to copy separate repo contents to project's folder

Reply:
Currently for this project I have 2 submodules repos and it will be very inconvenient to store them in a main repo like a folders...

Reply:
I still don't understand the purpose of this as this will not work per my explanation